librosa/display.py

```diff
@@ -950,6 +950,9 @@ def __decorate_axis(axis, ax_type):
         axis.set_label_text('')
         axis.set_ticks([])
 
+    else:
+        raise ParameterError('Unsupported axis type: {}'.format(ax_type))
+
```

Comment:
This was a missing error condition that upgrading the tests revealed (as an `xpass`). :tada: